src/util/dnd.cpp

```diff
@@ -193,7 +189,7 @@ QList<QFileInfo> DragAndDropHelper::dropEventFiles(
     qDebug() << "mimeData.hasText()" << mimeData.hasText() << mimeData.text();
 
     if (!mimeData.hasUrls() ||
-            (mimeData.hasText() && mimeData.text() == sourceIdentifier)) {
+        (mimeData.hasText() && mimeData.text() == sourceIdentifier)) {
         return QList<QFileInfo>();
     }
```

Comment:
Also here, you cannot distinguish the line break from  the conditional statement.
Is this a new setting in you format file? I did not remember these problems in the original auto format PR.

Reply:
Same here, indentation rules fail.